man-roxygen/optionalConnection.R

```diff
@@ -0,0 +1,3 @@
+#' @param optionalConnection   (optional) An object of type \code{connection} as created using the
```

Comment:
But the argument is called 'connection', not 'optionalConnection', right?

This should have flagged a warning in R check. Did you run R check?